### PR TITLE
added not-contains operator to filters

### PR DESCRIPTION
**Pull request by laurieboyes**
`!~` = not contains

### CODE REVIEW

**Comment by adambraimbridge** [comment]
Looks good to me. @wheresrhys ?

**Comment by wheresrhys** [comment]
:+1: aside from README

**Comment by wheresrhys** [comment]
cc @andygnewman you wanted something like this a while back didn't you?

File: lib/parser.js [hunk 1]
```diff
@@ -23,7 +23,7 @@ function merge(query, next) {
 }
 
 function parseFilter (filter) {
-	let filterConf = /^([^\!\~\?]*)(\!\?|\!=|=|>|<|\~|\?)([^=\~\?]*)$/.exec(filter);
+	let filterConf = /^([^\!\~\?]*)(\!\?|\!=|=|>|<|\~|\!\~|\?)([^=\~\?]*)$/.exec(filter);
```
Comment (wheresrhys) [inline lib/parser.js]: could you change this to

```
/^([^\!\~\?]*)(\!\?|\!=\!\~||=|>|<|\~|\?)([^=\~\?]*)$/
```

I may be being overly cautious, but I can never remember how greedy various regex matchers are, so am putting the 2-character ones at the beginning

Comment (wheresrhys) [inline lib/parser.js]: Ah no, am being overly cautious http://www.regexr.com/3csn4

In which case I suppose I prefer the way you did it, but would prefer if you've move `!?` to be next to `?` in that case, for consistency

Comment (laurieboyes) [inline lib/parser.js]: cool

**Pull request merged**